index.html

```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
+    <div class="blueprint-img col-6_md-hidden">
+      <img src="{{ "/images/blueprint.png" | relative_url }}" alt="open cloud blueprint" />
     </div>
   </div>
 </div>
+
 <div id="testimonials">
   <div>
     <div class="grid-center">
-      <div class="testimonial col-4_md-6_sm-12">
+      <div class="testimonial col-6_md-12">
         <h3>
           &ldquo;Crossplane has an opportunity to change the cloud industry as
-          we know it&rdquo;
+          we know it.&rdquo;
         </h3>
         <p>
           &ldquo;Our customers are increasingly looking for a way to deploy
           their code across multiple cloud environments. The choices available
-          today are too complex and vendor driven but with Crossplane the
+          today are too complex and vendor-driven, but with Crossplane, the
           ability to orchestrate clouds becomes simple. We look forward to
           collaborating with them on this vision and as the first complex app
           running on Crossplane.&rdquo;
         </p>
         <div class="signature">
-          <div class="name">Sid Sijbrandij</div>
-          <div class="title">CEO and Co-Founder of GitLab</div>
-          <a href="https://gitlab.com"><img src="{{ "/images/gitlab.svg" | relative_url }}"/></a>
+          <div class="exec-img">
+            <img srcset="./images/sid.jpg 1x, ./images/sid@2x.jpg 2x, ./images/sid@3x.jpg 3x" alt="Sid Sijbrandij" />
+          </div>
+          <div class="exec-info">
+            <div class="name">Sid Sijbrandij</div>
+            <div class="title">CEO and Co-Founder of GitLab</div>
+          </div>          
         </div>
       </div>
-      <div class="testimonial col-4_md-6_sm-12">
+      <div class="testimonial col-6_md-12">
         <h3>
-          &ldquo;We share the vision of having truly cloud agnostic, managed
-          services and Crossplane is an important step forward in this
-          direction&rdquo;
+          &ldquo;We share the vision of having truly cloud-agnostic managed
+          services.&rdquo;
         </h3>
         <p>
           &ldquo;Crossplane will make it possible for organizations to take
           deeper ownership of where they want to run their services and how they
           manage their application data. Kudos to the Upbound team for
           architecting and delivering this critical, open source and community
-          driven component of the cloud neutral future.&rdquo;
+          driven component of the cloud-neutral future.&rdquo;
         </p>
         <div class="signature">
-          <div class="name">Spencer Kimball</div>
-          <div class="title">CEO and Founder of Cockroach Labs</div>
-          <a href="https://www.cockroachlabs.com/"
-            ><img src="{{ "/images/cockroachlabs.svg" | relative_url }}"
-          /></a>
+          <div class="exec-img">
+            <img srcset="./images/spencer.jpg 1x, ./images/spencer@2x.jpg 2x, ./images/spencer@3x.jpg 3x" alt="Spencer Kimball" />
```

Comment:
I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper